<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>98</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Серия устройства</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>156</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Частота тактирования (МГц)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>107</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Модель устройства</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Частота тактирования переферийных устройств (МГц)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Серия 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Серия 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Серия 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Серия 99</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота 99</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модель 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модель 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модель 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модель 99</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота устройства 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота устройства 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота устройства 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота устройства 99</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>187</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать файл конфигурации .txt</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Задействованные интерфейсы</string>
    </property>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Многоразрядная шина</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SPI</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1-Wire</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>44</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ethernet</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>52</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>UART</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>I2C</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим работы портов GPIO / Режим подтяжки</string>
    </property>
    <widget class="QComboBox" name="comboBox_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Питание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Земля</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ввод</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вывод</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Питание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Земля</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ввод</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вывод</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Питание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Земля</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port C</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port D</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Питание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Земля</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port B</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ввод</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вывод</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port A</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ввод</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вывод</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл конфигурации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>196</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать файл конфигурации .Json</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="Console">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>400</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>467</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
